scheduler_failover_controller/failover/failover_controller.py

```diff
@@ -160,35 +193,75 @@ def is_scheduler_running(self, host):
             # todo: If there's more then one scheduler running this should kill off the other schedulers. MIGHT ALREADY BE HANDLED. DOUBLE CHECK.
 
             is_running = active_list_length > 0
+            if (is_running):
+                url = "http://" + \
+                    str(host) + ":" + str(conf.get("webserver",
+                                                   "web_server_port")) + "/health"
+                try:
+                    response = requests.request("GET", url)
+                    json_data = response. json()
+
+                    scheduler_status = json_data["scheduler"]["status"]
+                    if (str.lower(scheduler_status) == "healthy"):
+                        self.logger.info(
+                            "According to the webserver, is_running: " + str(is_running))
```

Comment:
Incorrect log. Should be ""According to the webserver, scheduler_status: " + str(scheduler_status))"